src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 spring.activemq.in-memory=true
-spring.activemq.pooled=false
\ No newline at end of file
+spring.activemq.pooled=false
```

Comment:
I thought it was enough that these were set in the test.properties file. But since these tests are using the SpringCoreDevOpsApplication.class, it is the application that needs them also.